tests/testthat/test-utils.R

```diff
@@ -36,3 +36,12 @@ test_that("respects httr verbose config", {
   withr::local_options(httr_config = config)
   expect_equal(httr2_verbosity(), 1)
 })
+
+test_that("sys_sleep has progress bar", {
+  withr::local_options(
+    cli.dynamic = FALSE,
```

Comment:
Why isn't this causing the progress bar to update once every 3s, like https://cli.r-lib.org/articles/progress-advanced.html#non-interactive-r-sessions says?

Reply:
Because the thread that provides the ticks is set up when cli is loaded, and then it is not updated: https://github.com/r-lib/cli/blob/01f932adafd58a59d596b745bf88359c5cb5a969/R/onload.R#L51-L66

Reply:
Hmmmm, any thoughts on how to test this then?

Reply:
Maybe create a function that formats the progress bar, and test that function, instead of "live" testing a progress bar.